build.js

```diff
@@ -92,6 +92,8 @@ function buildTestSuite(testsFolder, tags, all, exclude) {
         : list => tags.some(t => list.includes(t))
     const tests = readTestsFromDir(testsFolder)
         .filter(test => tags.length === 0 || (tagFilter(getTags(test)) !== exclude));
+    checkDuplicateNames(tests);
```

Comment:
We might want to check for dupes before applying `.filter(...)`.

Reply:
Is it ok then if I change `const tests` to `let tests`, and reassign it after the check with the filtered tests?

Reply:
Rr `allTests` and `filteredTests`.